src/util.ts

```diff
@@ -90,6 +90,44 @@ export async function showWorkspaceFoldersQuickPick(placeHolderString: string, t
     }
 }
 
+export async function showWarQuickPick(placeHolderString: string, telemetryProperties: TelemetryProperties): Promise<string> {
```

Comment:
We're trying to get rid of this generic util file. Please move this somewhere else (maybe into SiteTreeItem since that's the only place it's used? Also no need to export it in that case).